_data/navigation.yml

```
@@ -216,3 +216,5 @@
           name: governance
         - title: Website workgroup
           name: website-workgroup
+    - title: C++ Interoperability
+      url: /cxx-interop-workgroup/
```

To keep it consistent with the other pages, I suggest adding `sections` as well to the sidebar navigation.

```suggestion
    - title: C++ Interoperability
      url: /cxx-interop-workgroup/
      sections:
        - title: Charter
          name: charter
        - title: Membership
          name: membership
        - title: Communication
          name: communication
```